spec/supabase_js_v2.yml

```diff
@@ -5334,6 +5334,44 @@ pages:
             .download('folder/avatar1.png')
           ```
 
+  storage.from.publicDownload():
```

Comment:
We'll want to add/move these to the `feat/docs2.0` branch in these files:
- https://github.com/supabase/supabase/blob/feat/docs2.0/spec/supabase_js_v2_temp_new_shape.yml
- https://github.com/supabase/supabase/blob/feat/docs2.0/spec/common-client-libs-sections.json

cc: @MildTomato @saltcod